clap_complete/tests/testsuite/zsh.rs

```diff
@@ -154,10 +154,10 @@ fn complete() {
 
     let input = "exhaustive \t";
     let expected = snapbox::str![
-        r#"% exhaustive
-complete                                           -- Register shell completions for this program                     
-help                                               -- Print this message or the help of the given subcommand(s)       
-pacman    action  alias  value  quote  hint  last  --                                                                 "#
+        r#"% exhaustive[..]
+complete                                           -- Register shell completions for this program[..]
+help                                               -- Print this message or the help of the given subcommand(s)[..]
+pacman    action  alias  value  quote  hint  last  --  [..]"#
```

Comment:
Why do we need the `[..]` globs?  They make things more difficult

Reply:
I thought trailing spaces are hard to track. Anyway I will revert these changes.

Reply:
With snapshot testing, devs don't need to worry about the exact space counts.